modules/logging-loki-storage.adoc

```diff
@@ -0,0 +1,249 @@
+// Module is included in the following assemblies:
+// logging/logging-loki-ocp.adoc
+//
+:_content-type: PROCEDURE
+[id="logging-loki-storage_{context}"]
+= Loki log storage
+
+The Loki Operator supports link:https://aws.amazon.com/[AWS S3], as well as other S3 compatible object stores such as link:https://min.io/[Minio] and link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[OpenShift Data Foundation]. link:https://azure.microsoft.com[Azure], link:https://cloud.google.com/[GCS], and link:https://docs.openstack.org/swift/latest/[Swift] are also supported. 
+
+The recommended nomenclature for Loki storage is `logging-loki-<type>`. General secret creation format is as follows:
+//Generic secret creation
+include::snippets/logging-create-secret-snip.adoc[]
+
+You can verify a secret has been created by running the following command:
+
+[source,terminal]
+----
+oc get secrets
+----
+
+The table below displays the `type` values within the `LokiStack` CR for each storage provider. For more detailed information, consult the section for your storage provider.
+
+[options="header"]
+.Secret type quick reference
+|================================================
+| Storage provider          | Secret `type` value
+| AWS                       | s3
+| Azure                     | azure
+| Google Cloud              | gcs
+| Minio                     | s3
+| OpenShift Data Foundation | s3
+| Swift                     | swift
+|================================================
+
+== AWS S3
+
+.Prerequisites
+* Loki Operator deployed
+* Create a link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html[bucket] on AWS.
+
+.Procedure
+* Create an Object Storage secret with keys as follows:
+
+[source,terminal]
+----
+oc create secret generic logging-loki-s3 \
+  --from-literal=bucketnames="<BUCKET_NAME>" \
+  --from-literal=endpoint="<AWS_BUCKET_ENDPOINT>" \
+  --from-literal=access_key_id="<AWS_ACCESS_KEY_ID>" \
+  --from-literal=access_key_secret="<AWS_ACCESS_KEY_SECRET>" \
+  --from-literal=region="<AWS_REGION_OF_YOUR_BUCKET>"
+----
+
+** Suggested secret name: `logging-loki-s3`
+
+** Secret type: `s3`
+
+== Azure
+
+.Prerequisites
+* Loki Operator deployed
+* Create a link:https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction[bucket] on Azure.
+
+.Procedure
+* Create an Object Storage secret with keys as follows:
+
+[source,terminal]
+----
+oc create secret generic logging-loki-azure \
+  --from-literal=container="<AZURE_CONTAINER_NAME>" \
+  --from-literal=environment="<AZURE_ENVIRONMENT>" \
+  --from-literal=account_name="<AZURE_ACCOUNT_NAME>" \
+  --from-literal=account_key="<AZURE_ACCOUNT_KEY>"
+----
```

Comment:
Can you provide the example for AZURE_ENVIRONMENT as below?
# Azure Cloud environment. Supported values are: AzureGlobal, AzureChinaCloud,
# AzureGermanCloud, AzureUSGovernment.